docs/guides/modules/security/pages/audit-logs.adoc

```diff
@@ -1,22 +1,238 @@
+
+.. Permissions policy
++
+This policy grants the role permission to write objects to your S3 bucket (and any specified prefix).
++
+The minimum required access policy for the role is as follows:
++
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:PutObject",
+                "s3:GetObject",
+                "s3:ListBucket"
+            ],
+            "Resource": [
+                "arn:aws:s3:::<bucket-name>/*",
+                "arn:aws:s3:::<bucket-name>"
+            ]
+        }
+    ]
+}
+----
++
+Replace `<bucket-name>` with the name of your S3 bucket.
+
+. Set up an OIDC identity provider to allow CircleCI's OIDC tokens to be used for authentication.
+.. Navigate to the IAM Console in AWS.
+.. In the left navigation menu, select *Identity Providers*.
+.. Select *Add provider*.
+.. Choose *OpenID Connect* as the provider type.
+.. In the *Provider URL* field, specify `\https://oidc.circleci.com/org/<org-id>` (replace `<org-id>` with your CircleCI organization ID).
+.. In the *Audience* field, enter your organization ID.
+
+
+[#configure-audit-log-streaming]
+=== Configure Audit Log Streaming
+
+. Navigate to menu:Organization Settings[Security].
+. Scroll down to the **Audit Logs > Streaming audit logs** section and select btn:[Stream audit logs].
++
+image::guides:ROOT:stream-audit-logs.png[Stream audit logs section]
+
+. Confirm that you have completed the prerequisites by selecting btn:[Continue].
++
+image::guides:ROOT:confirm-prerequisites.png[Confirm prerequisites completion]
+
+. In the configuration dialog, specify the following fields:
+.. **Region**: The AWS region where your S3 bucket is located.
+.. **S3 Bucket Name**: The name of the S3 bucket where audit logs will be streamed.
+.. **Role ARN**: The Amazon Resource Name (ARN) for the IAM role that CircleCI will use to access your S3 bucket.
+
++
+image::guides:ROOT:connect-circleci-aws.png[Connect CircleCI to AWS]
+
+. Select btn:[Connect CircleCI to AWS].
+. When the connection to your S3 bucket is successfully validated, **Connected** with a green checkmark appears under **Status**.
++
+image::guides:ROOT:s3bucket-connected.png[S3 Connection successful]
+
+
+[#verification-and-testing]
+=== Verification and Testing
+
+After setting up audit log streaming:
+
+. Verify successful streaming by checking the **Last delivery** timestamp.
+. Confirm logs are appearing in your designated S3 bucket.
+. Test integrations with your SIEM or security monitoring tools.
+. Set up appropriate alerts for connection failures or delivery interruptions.
+
+
+[#managing-active-streams]
+=== Managing Active Streams
+
+Once configured, you can view and manage audit log streams.
```

Comment:
add something like, "The following options are available to you in the CircleCI web app"